vtysh/bgp_vty.c

```diff
@@ -2708,6 +2708,30 @@ is_ibgp_peer(const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor,
   return false;
 }
 
+void
+remove_parameters_invalid_for_ibgp_peer(const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor)
+{
+    if (ovs_bgp_neighbor->remove_private_as) {
```

Comment:
I suggest instead of operation with a pointer used with a counter operation.  
In this case "ovs_bgp_neighbor->remove_private_as" replace  "ovs_bgp_neighbor->n_remove_private_as"